### PR TITLE
[16.0][FIX] stock_cycle_count: removed some asserts for AccessError, stock_inventory: security groups

### PR DESCRIPTION
**Pull request by ArnauCForgeFlow**

### CODE REVIEW

File: stock_cycle_count/models/stock_inventory.py [hunk 1]
```diff
@@ -140,8 +140,7 @@ def _check_cycle_count_consistency(self):
                 raise ValidationError(
                     _(
                         "The Inventory Adjustment is inconsistent "
-                        "with the Cycle Count:\n%(message)s",
-                        message=msg,
+                        "with the Cycle Count:\n%s"
```
Comment (rousseldenis) [inline stock_cycle_count/models/stock_inventory.py]:
```suggestion
                        "with the Cycle Count:\n%(message)s", message=msg)
```

**Review by rousseldenis — APPROVED** [review]
Code review. Apart the comment on the translation

**Comment by LoisRForgeFlow** [comment]
For completeness, the incorrect migration of the tests was done here https://github.com/OCA/stock-logistics-warehouse/commit/b3185902a5917318c49a05025a4db75356ee6c11#diff-faaa3a465ad35d0326c4c4d5836110f073328b058004d938b1e664f88966d042R236. It passed unnoticed until now.

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks

**Pull request merged**